Apps/frontend/src/views/AmplitudeSlider.svelte

```diff
@@ -0,0 +1,27 @@
+<script>
+  import { amplitudeAdjustment } from "../stores.js";
+  import { MIN_AMPLITUDE, MAX_AMPLITUDE } from "../const.js";
+  export let onInput = (amplitude) => {
+    return amplitude;
+  };
+
+  export let channel;
+  // default amplification
+  let amplitude = 1.0;
+</script>
+
+<div>
+  <input
+    type="range"
+    bind:value={amplitude}
+    on:input={() => {
+      $amplitudeAdjustment[channel] = amplitude;
+      onInput(amplitude);
+    }}
+    min={MIN_AMPLITUDE}
+    max={MAX_AMPLITUDE}
+    step="0.01"
+    class="slider"
+    data-cy="amplitudeSlider-{channel}"
```

Comment:
Really like the way you approached the Slider component!
Think we should adapt OffsetSlider to have similar code structure, especially regarding testing.

Reply:
thanks!